src/publish.proj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Packaging.props))\Packaging.props" />
+  <Import Project="..\dir.targets" />
```

Comment:
The change looks fine but why are we only seeing it now?

Reply:
I didn't catch this initially because I didn't run a val build against CoreFx